grc/gui/Application.py

```diff
@@ -117,6 +117,7 @@ def _handle_action(self, action, *args):
         flow_graph = page.flow_graph if page else None
 
         def flow_graph_update(fg=flow_graph):
+            fg.update()
```

Comment:
This doubles the effort for model updates! I think its worth investigating the actual bug and enforce the proper namespace update within a single rewrite call

Reply:
Well I looked some days for a proper solution and the only one I found was to update twice.
Due to your hint now the bug in renew_namespace.